spark/src/main/scala/io/delta/tables/execution/DeltaTableOperations.scala

```diff
@@ -26,12 +26,14 @@ import org.apache.spark.sql.delta.commands.{DeltaGenerateCommand, DescribeDeltaD
 import org.apache.spark.sql.delta.util.AnalysisHelper
 import io.delta.tables.DeltaTable
```

Comment:
Nit: ```suggestion
import io.delta.tables.DeltaTable
import org.apache.hadoop.fs.Path

import org.apache.spark.sql.{functions, Column, DataFrame}
```

`org.apache.spark.sql....` should all be together

Reply:
Hmm doesn't really make sense because it should be internal -> external, but for some reason that's how it is other places too so ok